daemon/src/main/java/org/sonarlint/daemon/services/StandaloneSonarLintImpl.java

```diff
@@ -0,0 +1,111 @@
+/*
+ * SonarLint Daemon
+ * Copyright (C) 2009-2016 SonarSource SA
+ * mailto:contact AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+package org.sonarlint.daemon.services;
+
+import io.grpc.stub.StreamObserver;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.sonarsource.sonarlint.core.StandaloneSonarLintEngineImpl;
+import org.sonarsource.sonarlint.core.client.api.common.analysis.ClientInputFile;
+import org.sonarsource.sonarlint.core.client.api.standalone.StandaloneAnalysisConfiguration;
+import org.sonarsource.sonarlint.core.client.api.standalone.StandaloneGlobalConfiguration;
+import org.sonarsource.sonarlint.core.client.api.standalone.StandaloneGlobalConfiguration.Builder;
+import org.sonarsource.sonarlint.core.client.api.standalone.StandaloneSonarLintEngine;
+import org.sonarsource.sonarlint.daemon.proto.SonarlintDaemon.AnalysisReq;
+import org.sonarsource.sonarlint.daemon.proto.SonarlintDaemon.InputFile;
+import org.sonarsource.sonarlint.daemon.proto.SonarlintDaemon.Issue;
+import org.sonarsource.sonarlint.daemon.proto.SonarlintDaemon.LogEvent;
+import org.sonarsource.sonarlint.daemon.proto.SonarlintDaemon.StandaloneConfiguration;
+import org.sonarsource.sonarlint.daemon.proto.SonarlintDaemon.Void;
+import org.sonarsource.sonarlint.daemon.proto.StandaloneSonarLintGrpc.StandaloneSonarLint;
+
+import java.net.URL;
+import java.nio.charset.Charset;
+import java.nio.file.Paths;
+import java.util.LinkedList;
+import java.util.List;
+
+public class StandaloneSonarLintImpl extends AbstractSonarLint implements StandaloneSonarLint {
+  private static final Logger logger = LoggerFactory.getLogger(StandaloneSonarLintImpl.class);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename the "logger" logger to comply with the format "LOG(?:GER)?". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1312)